### PR TITLE
Add WordList with Context with saving to IndexedDB

### PR DESCRIPTION
**Pull request by irina060981**
It is the same changes as was in https://github.com/alpheios-project/components/pull/321

But I have fixed some errors, fixed tests, and upgraded it to current master branch.

It should be mereged with
https://github.com/alpheios-project/wordlist/pull/2
https://github.com/alpheios-project/data-models/pull/69

### CODE REVIEW

File: src/lib/queries/lexical-query.js [hunk 1]
```diff
@@ -123,7 +128,7 @@ export default class LexicalQuery extends Query {
         }
       })
       if (adapterTranslationRes.errors.length > 0) {
-        adapterTranslationRes.errors.forEach(error => console.error(error))
+        adapterTranslationRes.errors.forEach(error => console.error(error.messag4e))
```
Comment (balmas) [inline src/lib/queries/lexical-query.js]: think the 4 is a typo?

File: src/lib/selection/media/html-selector.js [hunk 2]
```diff
@@ -3,6 +3,8 @@ import { Constants, LanguageModelFactory } from 'alpheios-data-models'
 import TextSelector from '../text-selector'
 import MediaSelector from './media-selector'
 
+// import TextQuoteSelector from '@/lib/selection/w3c/text-quote-selector'
```
[the remark this answers is not in the document]
Comment (irina060981) [reply src/lib/selection/media/html-selector.js]: Removed

File: src/lib/selection/text-selector.js [hunk 3]
```diff
@@ -1,5 +1,7 @@
-import TextQuoteSelector from './w3c/text-quote-selector'
-import { LanguageModelFactory } from 'alpheios-data-models'
+// import TextQuoteSelector from './w3c/text-quote-selector'
```
[the remark this answers is not in the document]
Comment (irina060981) [reply src/lib/selection/text-selector.js]: Removed

File: src/lib/selection/w3c/text-quote-selector.js [hunk 4]
```diff
@@ -1,5 +1,21 @@
 /**
```
[the remark this answers is not in the document]
Comment (irina060981) [reply src/lib/selection/w3c/text-quote-selector.js]: Removed

**Pull request merged**